lucene/backward-codecs/src/java/org/apache/lucene/codecs/lucene70/Lucene70Codec.java

```diff
@@ -66,7 +67,7 @@ public PostingsFormat getPostingsFormatForField(String field) {
   private final DocValuesFormat docValuesFormat = new PerFieldDocValuesFormat() {
     @Override
     public DocValuesFormat getDocValuesFormatForField(String field) {
-      throw new IllegalStateException("This codec should only be used for reading, not writing");
+      return defaultDVFormat;
```

Comment:
Ahh this is because the DV updates must also be written with the old codec?

Reply:
I don't think it's a requirement, but I like using the old format better:
 - it's probably easier to debug, as all fields would be using the same format
 - given that PFDVFormat creates different files per format, forcing the new format would increase the number of files for old segments